services/dynamic-sidecar/tests/unit/test_api_container.py

```diff
@@ -0,0 +1,149 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
+import json
+from typing import Any, Dict, List
+
+import pytest
+from async_asgi_testclient import TestClient
+from simcore_service_dynamic_sidecar.storage import SharedStore
+
+
+@pytest.fixture
+def compose_spec() -> str:
+    return json.dumps(
+        {
+            "version": "3",
```

Comment:
side question: any limitations on the version we support? or is anything that docker-compose can swallow should work?

Reply:
We support all versions bigger then 1, meaning 2, 3 and future versions.